common/scripts/mesh-11s.sh

```diff
@@ -107,7 +107,7 @@ network={
     psk="$5"
     key_mgmt=SAE
     ieee80211w=2
-    mesh_fwding=0
```

Comment:
"mesh_fwding=0" configuration can be used with our wpa_supplicant.  If this is commented out, it breaks other projects which are using common scripts.

Reply:
af3cfe5 fixes this one.